.github/workflows/scripts/biome_disable_check.py

```
@@ -27,26 +26,25 @@
 import sys
 
 
-def has_eslint_disable(file_path):
+def has_biome_ignore(file_path):
```

Ruff (0.8.2)

Missing return type annotation for public function `has_biome_ignore`

(ANN201)